tests/pages/qanPage.js

```diff
@@ -0,0 +1,49 @@
+const I = actor();
+
+module.exports = {
+
+    fields : {
+        table: "//app-qan-table//div[@class='ps-content']",
+        filter: "//app-qan-filter//div[@class='ps-content']",
+        search: "//app-qan-search//input",
+        pagination: "//div[@class='pagination-tools']//ul[@class='ngx-pagination']",
+        pmmImage: "//footer/img",
+        pmmVersion: "//footer/small",
+        paginationArrow: "(//ul[@class='ngx-pagination']/li)[last()]",
+        addColumn: "//button[@class='add-column-btn']",
+        total: "//span[contains(text(), 'TOTAL')]",
+        columns: "//tbody//app-qan-table-header-cell",
+
```

Comment:
reason for this empty line?

Reply:
It was a mental note for me to mark that the number of the results section is yet to be tested.